### ISSUE TITLE
Pack into HTML, CSS, JavaScript File

### ISSUE THREAD

**Issue by khanfarhan10** [shreyaraj13/messenger-clone#1]
I'm looking for a backend for my chatbot and can take help from you shreya! Can you please make a single `index.html` file to perform the same?

**Comment by shreyaraj13** [comment]
This won't be possible Farhan because this app is made with create-react-app and cannot be made into an html file.

**Comment by khanfarhan10** [comment]
May i suggest flask?

**Comment by shreyaraj13** [comment]
Yes we can try building the backend with the help of flask to enhance the application.
